src/SprykerEco/Zed/UnzerApi/Persistence/Propel/Schema/spy_unzer_api.schema.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<database xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed" xsi:noNamespaceSchemaLocation="http://static.spryker.com/schema-01.xsd" namespace="Orm\Zed\UnzerApi\Persistence" package="src.Orm.Zed.UnzerApi.Persistence">
+
+    <table name="spy_payment_unzer_api_log" idMethod="native" allowPkInsert="true" isCrossRef="true">
```

Comment:
Currently all schemes that affect PSP have spy_payment_MODULE_* ... Do we really want to do it differently in this module?

Reply:
We discussed and agreed it with @kkicheglovspryker and @demkos  that this rule not be applied here. 
Because we want decrease migration effort for client which already use this solution on the project level.
So just ignore this comment.